packages/mobile/src/components/wait-for-download-drawer/WaitForDownloadDrawer.tsx

```diff
@@ -23,24 +26,32 @@ const messages = {
 }
 
 export const WaitForDownloadDrawer = () => {
+  const dispatch = useDispatch()
   const {
-    data: { contentId },
+    data: { trackIds },
     isOpen,
     onClose,
     onClosed
   } = useWaitForDownloadModal()
 
   const { spacing } = useTheme()
   const track = useSelector((state: CommonState) =>
-    getTrack(state, { id: contentId })
+    getTrack(state, { id: trackIds[0] })
```

Comment:
nit: seems a bit unsafe to assume the parent track ID is always gonna be in the first position. could pass in a second param to the drawer data. up to you.

Reply:
got u!